### ISSUE TITLE
Typo in pkg/minishift/openshift/service.go

### ISSUE THREAD

**Issue by agajdosi** [minishift/minishift#1215]
While working on #613 I have found another typo, having a typo week I guess. Instead of "exits" should be "exist".

```
[agajdosi@dhcp-24-156 ~]$ minishift openshift service list --namespace dreamteam
Namespace dreamteam doesn't exits
```

**Comment by hferentschik** [comment]
Create a pull request ;-)

**Comment by LalatenduMohanty** [comment]
Fixed in #1222
